lib/BackgroundJob/Tasks/CheckRequirementsTask.php

```diff
@@ -77,6 +77,7 @@ public function execute(FaceRecognitionContext $context) {
 			return false;
 		}
 
+		// Determine the system memory with some considerations.
 		$memory = MemoryLimits::getAvailableMemory();
```

Comment:
now that we have `$memory` and `$memorySetting`, consider naming them differently. For example, `$systemMemoryLimit` and `$userMemoryLimit`?

Reply:
I thought it but preferred to make the smallest patch, but now I do it. :wink: